doc/user/content/_index.md

```diff
@@ -69,4 +69,8 @@ primary_text="Get Started">}}
 
 ## Learn more
 
--   Check out the [**Materialize blog**](https://www.materialize.com/blog/) for the latest updates and Materialize deep-dives.
+- To find out about new features and improvements, see also the
```

Comment:
https://preview.materialize.com/materialize/32112/#learn-more

Note: We could actually put changelog as one of the options on the top nav bar of the docs ... next to Blog ... something to consider.

Reply:
Or the What's new box on the landing page.  Didn't in this pr since it might be better after cut a new version of the helm chart.